app/src/main/java/com/example/onsalestore/LaunchActivity.java

```diff
@@ -0,0 +1,52 @@
+package com.example.onsalestore;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+
+import com.example.onsale.R;
+
+public class LaunchActivity extends AppCompatActivity {
+
+    Button btnLogin, btnSign;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_launch);
+
+        btnLogin = findViewById(R.id.btnLogin);
+        btnSign = findViewById(R.id.btnSign);
+
+        btnLogin.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                goLoginActivity();
```

Comment:
Small nit, but naming should be very clear and contain all needed words.
```suggestion
                goToLoginActivity();
```

Reply:
Done